pkg/controller/pods.go

```diff
@@ -135,6 +135,7 @@ func (cont *AciController) podAdded(obj interface{}) {
 	pod := obj.(*v1.Pod)
 	cont.writeApicPod(pod)
 	cont.depPods.UpdatePodNoCallback(pod)
+	cont.erspanPolPods.UpdatePodNoCallback(pod)
```

Comment:
Why do you need to react to pod events? I thought ERSPAN would only react to podif events

Reply:
Yes, you are right. It reacts to podif events. Will clean it up.

Reply:
Tom, here this change is required to select the pods based on the labels  coming from ERSPAN CR. after selecting the pods,  using the pod info  podif cache can be used to get the required info